example-data/turtle/patientStatus.ttl

```diff
@@ -0,0 +1,30 @@
+@prefix : <http://purl.org/biosemantics-lumc/example-rdf/ern-common-data-elements/> .
+@prefix obo: <http://purl.obolibrary.org/obo/> .
+@prefix sio: <http://semanticscience.org/resource/> .
+@prefix ncit: <http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix snomedct: <http://purl.bioontology.org/ontology/SNOMEDCT/> .
+@prefix hl7: <http://purl.bioontology.org/ontology/HL7/> .
+
+# Example (alive)
+:person1
+  a obo:NCBITaxon_9606;
+  sio:SIO_000008 :status1 .
+
+:status1
+  a ncit:C25688;
+  sio:SIO_000300 obo:PATO_0001421 .
+
+# Example (dead)
```

Comment:
Can you split this example into two files? One file for `alive` and the other one for `dead`